aii-ks/src/main/pan/quattor/aii/ks/config.pan

```diff
@@ -84,6 +84,7 @@ variable AII_OSINSTALL_PROTOCOL = value('/system/aii/osinstall/ks/osinstall_prot
 #
 variable AII_OSINSTALL_ROOT ?= undef;
 variable AII_OSINSTALL_OS_VERSION ?= undef;
+variable DEBUG = debug(format('%s: AII_OSINSTALL_ROOT=%s, AII_OSINSTALL_OS_VERSION=%s', OBJECT,  to_string(AII_OSINSTALL_ROOT),  to_string(AII_OSINSTALL_OS_VERSION)));
```

Comment:
can you start the values on a new line? line looks a bit too long

Reply:
also, am i missing something or is this variable not used?

Reply:
@stdweird which variable? `DEBUG`? Sure it is just a way to trigger the `debug()` function... Do you have an alternative?